src/cluster_asm.c

```diff
@@ -1410,7 +1446,7 @@ void clusterSyncSlotsCommand(client *c) {
                                          task->source_offset - task->dest_offset,
                                          (int)ASM_PAUSE_WRITE_MAX_GAP_BYTES);
                     task->state = ASM_HANDOFF_PREP;
-                    clusterAsmOnEvent(task->slot_ranges, ASM_EVENT_HANDOFF_PREP, NULL);
+                    clusterAsmOnEvent(task->id, ASM_EVENT_HANDOFF_PREP, task->slot_ranges);
```

Comment:
we should pass `task->slot_ranges` to indicate which slots we'd like to stop writes. Plugin will not use this now but just in case. Similar for ASM_EVENT_MIGRATE_STARTED event.